notebooks/22_special_manifolds_Klein_bottle.ipynb

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
<p>Very nice! change the name of the notebook to 22_special_manifolds_klein_bottle.ipynb</p><p>(to uppercase letters in filenames)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1820/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>